tools/coreograph/coreograph.xml

```diff
@@ -0,0 +1,100 @@
+<tool id="unet_coreograph" name="UNetCoreograph" version="@TOOL_VERSION@+galaxy@VERSION_SUFFIX@" profile="19.01">
+    <description>Coreograph uses UNet, a deep learning model, to identify complete/incomplete tissue cores on a tissue microarray. It has been trained on 9 TMA slides of different sizes and tissue types.</description>
```

Comment:
Too long for a tool description. This will appear in the toolbox and might clutter the box.